### ISSUE TITLE
cc-oci-runtime fails to parse a json file when running with docker 1.13.1 

### ISSUE THREAD

**Issue by chavafg** [01org/cc-oci-runtime#691]
Tried to run clear containers with docker 1.13.1. When running `docker run ` from the command line, the containers seem to work, but I see this failure in the logs: 

```
$ docker run -ti ubuntu bash 
```
cc-oci-runtime.log:
```
2017-02-22T21:39:00.083954Z:6681:cc-oci-runtime:debug:communicating with proxy
2017-02-22T21:39:00.084000Z:6681:cc-oci-runtime:debug:sending message (length 458) to proxy socket
2017-02-22T21:39:00.084038Z:6681:cc-oci-runtime:debug:writing message data to proxy socket: {"id":"hyper","data":{"hyperName":"newcontainer","data":{"id":"a52c2cedfbcce866d6e56ccdf2234946bc1808724a0e0a62544ee718011e2c56","rootfs":"","image":"","restartPolicy":"never","initialize":false,"process":{"terminal":false,"stdio":1,"stderr":2,"workdir":"/","user":"root","args":["env"],"envs":[{"value":"/root","env":"HOME"},{"value":"/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin","env":"PATH"},{"value":"a52c2cedfbcc","env":"HOSTNAME"}]}}}}
2017-02-22T21:39:00.197104Z:6681:cc-oci-runtime:debug:proxy msg length: 16
2017-02-22T21:39:00.197217Z:6681:cc-oci-runtime:debug:message read from proxy socket: {"success":true}
2017-02-22T21:39:00.197317Z:6681:cc-oci-runtime:debug:msg received: {"success":true}
2017-02-22T21:39:00.197352Z:6681:cc-oci-runtime:debug:disconnecting from proxy
2017-02-22T21:39:00.197729Z:6681:cc-oci-runtime:debug:created state file /var/run/cc-oci-runtime/a52c2cedfbcce866d6e56ccdf2234946bc1808724a0e0a62544ee718011e2c56/state.json
2017-02-22T21:39:00.394025Z:6682:cc-oci-runtime:debug:called as: /usr/local/bin/cc-oci-runtime kill --all a52c2cedfbcce866d6e56ccdf2234946bc1808724a0e0a62544ee718011e2c56 SIGKILL
2017-02-22T21:39:00.394385Z:6682:cc-oci-runtime:debug:unable to parse '/var/run/cc-oci-runtime/--all/state.json'
2017-02-22T21:39:00.394424Z:6682:cc-oci-runtime:debug:Error parsing '/var/run/cc-oci-runtime/--all/state.json': Failed to open file '/var/run/cc-oci-runtime/--all/state.json': No such file or directory
2017-02-22T21:39:00.394463Z:6682:cc-oci-runtime:critical:failed to parse json file: /var/run/cc-oci-runtime/--all/state.json
2017-02-22T21:39:00.394510Z:6682:cc-oci-runtime:critical:failed to read state file for container --all
```

**Comment by chavafg** [comment]
Reproduced the issue on CentOS 7 and Fedora 25.

Also, the `integration/docker` tests are getting hanged when the containers are being removed.

**Comment by jodh-intel** [comment]
See #693.

**Comment by devimc** [comment]
fixed in #776
